haskell-stack/README.md

```diff
@@ -0,0 +1,28 @@
+# Haskell-stack dockerfile
+
+Creates an image with [`stack`](http://docs.haskellstack.org/en/stable/README/) installed.
+
+## Build
+
+To build, run:
+
+```
+cd haskell-stack
+docker build -t advancedtelematic/haskell-stack:1.0.2 .
+```
+
+## Usage
+
+This needs `stack setup` to be run and persisted between calls, so first up run the following:
+
+```
+docker create --name stack-cache -v /root/.stack tianon/true /bin/true 2>/dev/null || true'
```

Comment:
Why would we need to have persistence volume? what happens if the container writes on its own file-system?